test/tasks/ods-package-image_test.go

```diff
@@ -209,3 +175,49 @@ func getDockerImageTag(t *testing.T, ctxt *tasktesting.TaskRunContext, wsDir str
 	}
 	return fmt.Sprintf("localhost:5000/%s/%s:%s", ctxt.Namespace, ctxt.ODS.Component, sha)
 }
+
+func generateArtifacts(t *testing.T, ctxt *tasktesting.TaskRunContext, tag string, wsDir string) {
+	t.Logf("Generating artifacts for image %s", tag)
+	t.Logf("Generating image artifact")
+	err := generateImageArtifact(ctxt, tag, wsDir)
+	if err != nil {
+		t.Fatalf("could not create image artifact: %s", err)
+	}
+	t.Logf("Generating image SBOM artifact")
+	err = generateImageSBOMArtifact(ctxt, wsDir)
+	if err != nil {
+		t.Fatalf("could not create image SBOM artifact: %s", err)
+	}
+}
+
+func generateImageArtifact(ctxt *tasktesting.TaskRunContext, tag string, wsDir string) error {
+	sha, err := getTrimmedFileContent(filepath.Join(wsDir, ".ods/git-commit-sha"))
+	if err != nil {
+		return fmt.Errorf("could not read git-commit-sha: %s", err)
+	}
+	ia := artifact.Image{
+		Ref:        tag,
+		Registry:   "kind-registry.kind:5000",
+		Repository: ctxt.Namespace,
+		Name:       ctxt.ODS.Component,
+		Tag:        sha,
+		Digest:     "abc",
+	}
+	imageArtifactFilename := fmt.Sprintf("%s.json", ctxt.ODS.Component)
+	err = pipelinectxt.WriteJsonArtifact(ia, filepath.Join(wsDir, pipelinectxt.ImageDigestsPath), imageArtifactFilename)
+	return err
+}
+
+func generateImageSBOMArtifact(ctxt *tasktesting.TaskRunContext, wsDir string) error {
```

Comment:
I think for this function, but also for the one above, it would be easier to pass in `t *testing.T` and simply call `t.Fatal(err)` instead of returning the error.